CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Table of Contents
+
+- [v0.1.0](#v010)
+
+
+## v0.1.0
```

Comment:
I think it would be helpful to include API versions in here somehow as well (v1alpha1).

Reply:
Do you mean in the changelog headers or in the Git tag itself?

Reply:
I think probably just the changelog unless we can also add a v1alpha1 tag for the same commit without breaking go modules.